.justfile

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env just --justfile
 
-set dotenv-load := true
```

Comment:
assuming this has been removed in favor or `dotenv_flow`?

Reply:
Yes. If this were kept, the `.env` variables would be passed to the Graph and the graph wouldn't use the `.env.*` files as the environment variables would already be specified.